modules/secrets-manager/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>secrets-manager-sdk</artifactId>
         <groupId>com.ibm.cloud</groupId>
-        <version>2.0.15</version>
+        <version>2.0.16</version>
```

Comment:
This change should be removed. Publication automation bumps version automatically.